mmv1/third_party/terraform/services/resourcemanager/data_source_google_organization_iam_custom_role_test.go

```diff
@@ -0,0 +1,59 @@
+package resourcemanager_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccDataSourceGoogleOrganizationIamCustomRole_basic(t *testing.T) {
+	t.Parallel()
+
+	orgId := envvar.GetTestOrgFromEnv(t)
+	roleId := "tfIamCustomRole" + acctest.RandString(t, 10)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckGoogleOrganizationIamCustomRoleConfig(orgId, roleId),
+				Check: resource.ComposeTestCheckFunc(
+					acctest.CheckDataSourceStateMatchesResourceState(
+						"data.google_organization_iam_custom_role.this",
+						"google_organization_iam_custom_role.this",
+					),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckGoogleOrganizationIamCustomRoleConfig(orgId string, roleId string) string {
+	return fmt.Sprintf(`
+locals {
+  org_id  = "%s"
+  role_id = "%s"
+}
+
+resource "google_organization_iam_custom_role" "this" {
+  org_id      = local.org_id
+  role_id     = local.role_id
+  title       = "Terraform Test"
+
+  permissions = [
+	"iam.roles.create",
+	"iam.roles.delete",
+    "iam.roles.list",
+  ]
+}
```

Comment:
```suggestion
resource "google_organization_iam_custom_role" "this" {
  org_id      = "%s"
  role_id     = "%s"
  title       = "Terraform Test"

  permissions = [
    "iam.roles.create",
    "iam.roles.delete",
    "iam.roles.list",
  ]
}
```
nit-picks

Reply:
Thanks for the input.
I've adjusted the code accordingly.

https://github.com/GoogleCloudPlatform/magic-modules/compare/e0cd602294776b4d56f3f9e244d350d99fab7166..8a51ec79dbb87636069134afddbd933f15adb5a5

Please, have a look again.